src/main/java/de/rub/nds/scanner/core/report/ScanReport.java

```diff
@@ -38,31 +39,13 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import java.util.Observable;
 import java.util.Set;
 import java.util.stream.Collectors;
 
-@JsonIncludeProperties({
-    "results",
-    "extractedValues",
-    "guidelineReports",
-    "scoreReport",
-    "probePerformanceData",
-    "performedConnections",
-    "scanStartTime",
-    "scanEndTime"
-})
-@JsonPropertyOrder({
-    "scanStartTime",
-    "scanEndTime",
-    "performedConnections",
-    "results",
-    "extractedValues",
-    "guidelineReports",
-    "scoreReport",
-    "probePerformanceData"
-})
-public abstract class ScanReport extends Observable {
+public abstract class ScanReport {
```

Comment:
maybe we also need to serialize the class here as the client report needs to be differentiated